cmd/cloud-endpoints-controller/main.go

```diff
@@ -126,84 +128,44 @@ func sync(parent *CloudEndpoint, children *CloudEndpointControllerRequestChildre
 	}
 
 	if currState == StateEndpointCreatePending {
+		log.Printf("[INFO][%s] Create pending", parent.Name)
 		var target string
-		var openAPISpec string
+		var openAPISpecTemplate string
+		var finalOpenAPISpec string
```

Comment:
Is there a reason this is declared explicitly as a var? I only see it being set once.